code/modules/mapping/reader.dm

```diff
@@ -60,25 +72,37 @@
 	placeOnTop = FALSE as num,
 	orientation = SOUTH as num,
 	annihilate_tiles = FALSE,
-	z_lower = -INFINITY as num,
-	z_upper = INFINITY as num
+	crop_relative_to_game_world = TRUE
 	)
-	var/datum/parsed_map/parsed = new(dmm_file, x_lower, x_upper, y_lower, y_upper, z_lower, z_upper, measureOnly)
```

Comment:
with orientation there's no way to do this without breaking, if people want to crop maps and math it out manually they can imeplement it custom instead of use load_map().